tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc

```diff
@@ -1036,6 +1036,7 @@ TEST_F( EnergyPlusFixture, UnitarySystem_GetInput ) {
 	Schedule( 1 ).CurrentValue = 1.0;
 	DataGlobals::BeginEnvrnFlag = true;
 	DataEnvironment::StdRhoAir = PsyRhoAirFnPbTdbW( 101325.0, 20.0, 0.0 ); // initialize RhoAir
+	Node( InletNode ).MassFlowRateMaxAvail = UnitarySystem( 1 ).MaxCoolAirVolFlow * StdRhoAir;
```

Comment:
@rraustad What if the heating or no load flow rate is greater than MaxCoolAirVolFlow?